htdocs/public/payment/newpayment.php

```diff
@@ -1543,7 +1557,7 @@ class="stripe-button"
 	    <div id="card-element">
 	    <!-- a Stripe Element will be inserted here. -->
 	    </div>
-
+      <input name="savesource" type="checkbox"> '.$langs->trans("SaveSource").'
```

Comment:
What is this box for ? What happen if user check this box ?

Reply:
with rgpd, we can only save for later source/card with an explicit customer's agreement
i will add the code to use it